### DISCUSSION TITLE
I haven't receibe my education pack after 5 days

### DISCUSSION THREAD

**Discussion by GarotoCowboy** [community/community#131803] [GitHub Education] [answered]
### Hi everyone,

 Hi, I signed up for the educational package on the 5th, I was approved but I still haven't received the benefits after 5 business days.
can you help me?
![Iprint screen](https://github.com/community/community/assets/34407863/4ae842bf-8b14-40eb-a88f-cc6a01b61032)

**Comment by davevad93** [comment, marked as answer]
Hi @GarotoCowboy ,

Usually it takes a few days, but sometimes it can take even a couple of weeks, the waiting time may vary depending on different factors such as the volume of applications. When the benefits become available the status label will turn purple and change to:

`Congrats! Your request for @GarotoCowboy was approved and your benefits are now available`

**Reply by Arlene-1998** [reply]
It has been 5 months since my approval, far longer than couple of weeks

**Reply by davevad93** [reply]
Hi @Arlene-1998 , 5 months is a very long time and it's not normal. If you haven't received any responses about your application, try with rising a ticket [here](https://support.github.com/contact/account), select `Other` and explain you issue. There is also the [GitHub Education support](https://support.github.com/contact/education) but it redirects here in the discussions so I advise you to also open a ticket where I told you.
